### ISSUE TITLE
ansible-galaxy install fails when running on CentOS/RHEL 7 and only for geerlingguy.php!

### ISSUE THREAD

**Issue by jonpugh** [ansible/ansible#15251]
##### ISSUE TYPE
- Bug Report
##### ANSIBLE VERSION

```
ansible 1.9.4
  configured module search path = None
```
##### CONFIGURATION

None
##### OS / ENVIRONMENT

CentOS 7.  A friend saw it happen on RHEL 7 as well.
##### SUMMARY

This is a very strange one, I believe it only happens on CentOS/RHEL 7 machines.

I can download plenty of roles from ansible galaxy, but if I try to download geerlingguy.php, ansible-galaxy errors out.
##### STEPS TO REPRODUCE

```
[vagrant@local ~]$ ansible-galaxy install geerlingguy.php 
```
##### EXPECTED RESULTS

Something like I get with other roles:

```
[vagrant@local ~]$ sudo ansible-galaxy install geerlingguy.java
- downloading role 'java', owned by geerlingguy
- downloading role from https://github.com/geerlingguy/ansible-role-java/archive/1.3.0.tar.gz
- extracting geerlingguy.java to /etc/ansible/roles/geerlingguy.java
- geerlingguy.java was installed successfully
```
##### ACTUAL RESULTS

```
[vagrant@local ~]$ sudo ansible-galaxy install geerlingguy.php
- downloading role 'php', owned by geerlingguy
https://galaxy.ansible.com/roles/432/versions/?page=2&page_size=50
Traceback (most recent call last):
  File "/bin/ansible-galaxy", line 959, in <module>
    main()
  File "/bin/ansible-galaxy", line 953, in main
    fn(args, options, parser)
  File "/bin/ansible-galaxy", line 819, in execute_install
    if len(role_versions) > 0:
TypeError: object of type 'NoneType' has no len()
```

**Comment by chouseknecht** [comment]
This seems to be specific to 1.9.4.  Installing geerlingguy.php with 1.9.5 works as expected.

```
$ ansible --version
ansible 1.9.5 (stable-1.9 331f6ba52e) last updated 2016/04/03 03:07:17 (GMT -400)
  lib/ansible/modules/core:  not found - use git submodule update --init lib/ansible/modules/core
  lib/ansible/modules/extras:  not found - use git submodule update --init lib/ansible/modules/extras
  configured module search path = None
```

```
$ ansible-galaxy install geerlingguy.php
- downloading role 'php', owned by geerlingguy
https://galaxy.ansible.com/api/v1/roles/432/versions/?page=2&page_size=50
- downloading role from https://github.com/geerlingguy/ansible-role-php/archive/2.0.2.tar.gz
- extracting geerlingguy.php to ./geerlingguy.php
- geerlingguy.php was installed successfully
```

**Comment by jctanner** [comment]
Hi!

Thanks very much for your submission to Ansible.  It sincerely means a lot to us. 

Testing this on the version you have specified, we cannot reproduce this problem.

Our reasons and evidence for this are:
- As noted by @chouseknecht , this appears to be fixed in ansible 1.9.5, and was also addressed by 
  https://github.com/ansible/galaxy-issues/issues/134.

And we suggest that:
- Please try updating to at least ansible 1.9.5

As such, we are most likely going to close this ticket.  Should you disagree with this conclusion, you may also wish to join our mailing lists, which are very active:
- https://groups.google.com/forum/#!forum/ansible-project - for user questions, tips, and tricks
- https://groups.google.com/forum/#!forum/ansible-devel - for strategy, future planning, and questions about writing code

Thank you once again for this and your interest in Ansible!
